account_allowance_adjust_transaction_unit_test.go

```diff
@@ -0,0 +1,206 @@
+//go:build all || unit
+// +build all unit
+
+package hedera
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+/*-
+ *
+ * Hedera Go SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+func TestUnitAccountAllowanceAdjustTransactionGet(t *testing.T) {
+	key, err := PrivateKeyGenerate()
+	key2, err := PrivateKeyGenerate()
+	require.NoError(t, err)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏In case we got an error for the first key but not for the second this require will pass instead of failing. Probably we need another require right after the first key creation?